glu.go

```diff
@@ -59,7 +65,30 @@ func (r *Registry) Run(ctx context.Context) error {
 		return r.runOnce(ctx)
 	}
 
-	var group errgroup.Group
+	var (
+		group errgroup.Group
+		srv   = http.Server{
+			Addr:    ":8080", // TODO: make configurable
+			Handler: r.server,
+		}
+	)
+
+	group.Go(func() error {
+		shutdownCtx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+		defer cancel()
+
+		<-ctx.Done()
```

Comment:
Just need to reorder, otherwise, the 15 seconds still start from when glu starts.
```suggestion
		<-ctx.Done()
		
		shutdownCtx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
		defer cancel()
```

Reply:
ah dang good catch